generators/src/filepaths.ts

```diff
@@ -140,6 +140,8 @@ function filepathsFn(options: FilepathsOptions = {}): Fig.Generator {
   };
 
   return {
+    [Symbol.generatorTag]: "filepathsTemplate",
```

Comment:
NOTE: adding symbols properties to our objects makes a strong requirement for those creating tools relying on specs (e.g. AE, Dashboard) and running generators. They are basically a protocol that should be implemented.
e.g.
```ts
const generator: Fig.Generator = {
  [Symbol.generatorTag]: "name"
}
// if Symbol.generatorTag is not defined in the environment (see the ref PR in AE to understand how it is defined)
// the objects assumes the following shape
const generator: Fig.Generator = {
  undefined: "name"
}
// which means that if I try to access `undefined` I get name
generator[undefined] // "name"
generator["undefPropertyName"] // "name"
```